### PR TITLE
Update the .yaml files to v1 API, ...

### PR DESCRIPTION
**Pull request by amygdala**
... add date timestamps to the bq logging info, add additional field filters to the tweets processing (until the bq schema is updated), README update.

### CODE REVIEW

File: pubsub/bigquery-controller.yaml [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
-                value: xxxx
-    # Important: these labels need to match the selector above
-    # The api server enforces this constraint.
-    labels:
-      name: bigquery-controller
+  name: bigquery-controller
+spec:
+  replicas: 1
+  selector:
+    name: bigquery-controller
```
Comment (jeffmendoza) [inline pubsub/bigquery-controller.yaml]: The selector is redundant now, and is picked up automatically from the template labels. Unless you wanted specific differences.

File: pubsub/bigquery-controller.yaml [hunk 2]
```diff
@@ -1,38 +1,32 @@
...
+  template:
+    metadata:
+      labels:
+        name: bigquery-controller
+    spec:
+      containers:
+      - image: gcr.io/your-project/your-image-tag
+        name: bigquery
```
Comment (jeffmendoza) [inline pubsub/bigquery-controller.yaml]: Tag is the thing after the colon, usually the version.

File: pubsub/twitter-stream.yaml [hunk 3]
```diff
@@ -1,42 +1,36 @@
...
+  labels:
+    name: twitter-stream
+  name: twitter-stream
+spec:
   replicas: 1
-  # replicaSelector identifies the set of Pods that this
-  # replicaController is responsible for managing
-  replicaSelector:
+  selector:
     name: twitter-stream
```
Comment (jeffmendoza) [inline pubsub/twitter-stream.yaml]: same notes here

File: redis/bigquery-controller.yaml [hunk 4]
```diff
@@ -1,37 +1,31 @@
...
+  labels:
+    name: bigquery-controller
+  name: bigquery-controller
+spec:
   replicas: 2
-  # replicaSelector identifies the set of Pods that this
-  # replicaController is responsible for managing
-  replicaSelector:
+  selector:
     name: bigquery-controller
```
Comment (jeffmendoza) [inline redis/bigquery-controller.yaml]: same notes here

File: redis/redis-master-service.yaml [hunk 5]
```diff
@@ -1,15 +1,13 @@
...
-labels:
+metadata:
+  labels:
+    name: redis-master
   name: redismaster
+spec:
+  ports:
+    # the port that this service should serve on
+  - port: 6379
+    targetPort: 6379
```
Comment (jeffmendoza) [inline redis/redis-master-service.yaml]: Target port is redundant if same as port

File: redis/redis-master.yaml [hunk 6]
```diff
@@ -1,24 +1,20 @@
...
-            # name must match the volume name below
-          - name: redis-persistent-storage
-            # mount path within the container
-            mountPath: /data/redis
-    volumes:
-      - name: redis-persistent-storage
-        source:
-          emptyDir: {}
-labels:
+apiVersion: v1
```
Comment (jeffmendoza) [inline redis/redis-master.yaml]: Does this need to be an RC if it will always be 1 pod?

File: redis/twitter-stream.yaml [hunk 7]
```diff
@@ -1,41 +1,35 @@
...
+  labels:
+    name: twitter-stream
+  name: twitter-stream
+spec:
   replicas: 1
-  # replicaSelector identifies the set of Pods that this
-  # replicaController is responsible for managing
-  replicaSelector:
+  selector:
     name: twitter-stream
```
Comment (jeffmendoza) [inline redis/twitter-stream.yaml]: same notes about selector/tag

**Comment by jeffmendoza** [comment]
Another nit =) I was told that name: should be first in metadata: (before labels:). https://github.com/GoogleCloudPlatform/kubernetes/pull/9086#discussion_r31484426

File: redis/redis-master.yaml (hunk 6 above)
Comment (amygdala) [inline redis/redis-master.yaml]: It doesn't need to, but this is what the gb app does, making the case that you get the auto-restarts this way (it's also made clear that this redis installation is not persistent).
I was thinking to show the same-- I think I'll leave it this way, but add more explication.

File: redis/redis-master-service.yaml (hunk 5 above)
Comment (amygdala) [inline redis/redis-master-service.yaml]: Thanks, yeah.  Was thinking to leave it even so, so they could see there was a targetPort setting. Will add a bit more explanation if I do that.

**Comment by jeffmendoza** [comment]
LGTM

**Pull request merged**